main.html

```diff
@@ -1,77 +1,255 @@
+</h2>
+
+<p>
+  There are many benefits to collaborating via InnerSource.
+  First and foremost, <strong>the guest team gets their feature</strong> request when they need it without the long-term burden of maintaining it.
+  When the guest team would have spent time on the feature regardless (whether through <em>Workaround</em> or <em>Escalate</em>), InnerSource provides a way to use that time towards developing the feature in a place that's centrally-maintained and shared.
+</p>
+
+<p>
+  While that result is the shining benefit of InnerSource, another result that's easy to overlook is the fact that <strong>the host team gets their feature, too</strong>.
+  Recall that, as part of the InnerSource process, the product owner on the host team agrees from the outset that the requested feature is good and desirable.
+  InnerSource allows the host team to receive help in creating a better product for its consumers!
+</p>
+
+<p>
+  InnerSourcing provides the host team a scalable strategy for fulfilling the varying amounts of features requested from its many consumers.
+  Given the fixed capacity of the full-time members of the host team, it's likely that at times the combined business roadmaps of their consumers will require very (or even unreasonably) large amounts of work to be done in the host team's product(s).
+  Without InnerSource, this situation easily leads to a stressed, overworked team dealing with many feature requests escalated to its leaders.
+  However, if the host team operates via InnerSource, the engineering resources required to build those features will appear in-proportion to their importance in the form of guest contributors.
+  InnerSource becomes a force multiplier that allows a host team to temporarily act larger than its actual size during times of high demand.
+  When the demand has ended, the team throughput scales back to normal levels, all without any micromanagement of team headcount or work items.
+</p>
+
+<p>
+  Beyond the raw work that the host team is able to accomplish in its system, regular InnerSource contributions give the host team better requirements and prioritization alignment with all of its consumers.
+  A host team can do its best requirements gathering on the work it produces, but when the consumer itself is the one submitting the work the chances are much higher that the resulting change is aligned to just what the consumer needs.
+  Sure, it's only one single guest team submitting the change, but that team is likely representative of many other consumers and their needs.
+</p>
+
+<p>
+  In addition to this alignment, there is also a grass-roots evangelism that takes hold in many contributors.
+  Having made a code contribution, they feel a stake in the host team project.
+  It's something that they recommend to their colleagues or new teams that they join.
+  They understand the project better and are able to answer questions about it to others, relieving the host team of some of that burden.
```

Comment:
This also helps eliminate the tribal nature of siold company cultures.  Might be worth a mention.